Sakura/core/handlers.py

```
@@ -0,0 +1,245 @@
+
+
+async def error_handler(update: object, context: ContextTypes.DEFAULT_TYPE) -> None:
+    """Handle errors"""
+    log_with_user_info("ERROR", f"Exception while handling an update: {context.error}", extract_user_info(update.message))
+
+    # Try to extract user info if update has a message
+    if hasattr(update, 'message') and update.message:
+        try:
+            user_info = extract_user_info(update.message)
+            log_with_user_info("ERROR", f"💥 Exception occurred: {context.error}", user_info)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)